### ISSUE TITLE
Failure when using virtualenv

### ISSUE THREAD

**Issue by OnceUponALoop** [codeflows/vaulty#7]
Working on an environment that does not have ansible locally installed and is rather relying on Ansible from a Python virtualenv - it seems vaulty is unable to find `ansible-vault`.

I've tried defining the python interpreter via python.pythonPath but that didn't seem to make a difference in vaulty failures.
```
{
    "python.pythonPath": ".venv/bin/python"
}
```

Output log fails to indicate where the issue is coming from

```
vaulty activated
Attempting to delegate to vaulty:/home/demo/vm_orchestration/group_vars/win_all/vault.yml#0
Found 7 ansible.cfg file(s) in workspace: file:///home/demo/vm_orchestration/ansible.cfg,file:///home/demo/vm_orchestration/.venv/lib/python2.7/site-packages/ansible_test/_data/ansible.cfg,file:///home/demo/vm_orchestration/.venv/lib64/python2.7/site-packages/ansible_test/_data/ansible.cfg,file:///home/demo/vm_orchestration/.venv/lib/python2.7/site-packages/ansible/galaxy/data/apb/tests/ansible.cfg,file:///home/demo/vm_orchestration/.venv/lib/python2.7/site-packages/ansible/galaxy/data/container/tests/ansible.cfg,file:///home/demo/vm_orchestration/.venv/lib64/python2.7/site-packages/ansible/galaxy/data/apb/tests/ansible.cfg,file:///home/demo/vm_orchestration/.venv/lib64/python2.7/site-packages/ansible/galaxy/data/container/tests/ansible.cfg
1 of these are located in parent directories of the Vault file: file:///home/demo/vm_orchestration/ansible.cfg
Found exactly one configuration file: file:///home/demo/vm_orchestration/ansible.cfg
Found vault_password_file in /home/demo/vm_orchestration/ansible.cfg, resolved ./.vault_pass to /home/demo/vm_orchestration/.vault_pass
Decrypting vault with arguments "--vault-password-file=/home/demo/vm_orchestration/.vault_pass --output=- decrypt /home/demo/vm_orchestration/group_vars/win_all/vault.yml"
```

I'm not sure how to debug it further - if you can provide some pointers I can probably produce better debugging info :|

I should add that manually running with the parameters above (in the right order) works just fine and decrypts the vault as expected.

**Comment by codeflows** [comment]
Hi @OnceUponALoop, and thanks for the bug report!

I recently fixed #9 which was caused by parameters being passed to `ansible-vault` in the wrong order. This fix is included in Vaulty 0.2.1, I was wondering if you could try it out so that we can see if it's the same problem you had?

**Comment by codeflows** [comment]
Just released 0.3.0 with some additional fixes; I'm closing this issue for now but please let me know if the new release doesn't fix your issue. Thanks!
